src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalpostClient.kt

```diff
@@ -0,0 +1,77 @@
+package no.nav.tilleggsstonader.sak.journalføring
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.tilleggsstonader.kontrakter.dokarkiv.OppdaterJournalpostRequest
+import no.nav.tilleggsstonader.kontrakter.dokarkiv.OppdaterJournalpostResponse
+import no.nav.tilleggsstonader.kontrakter.felles.ObjectMapperProvider.objectMapper
+import no.nav.tilleggsstonader.kontrakter.journalpost.Dokumentvariantformat
+import no.nav.tilleggsstonader.kontrakter.journalpost.Journalpost
+import no.nav.tilleggsstonader.libs.http.client.AbstractRestClient
+import no.nav.tilleggsstonader.libs.log.NavHttpHeaders
+import no.nav.tilleggsstonader.sak.opplysninger.søknad.domain.SøknadBarnetilsyn
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.util.UriComponentsBuilder
+import java.net.URI
+
+@Component
+class JournalpostClient(
+    @Value("\${clients.integrasjoner.uri}") private val integrasjonerBaseUrl: URI,
+    @Qualifier("azure") restTemplate: RestTemplate,
+) : AbstractRestClient(restTemplate) {
+
+    fun hentJournalpost(journalpostId: String): Journalpost =
+        getForEntity<Journalpost>(journalpostUri.toString(), uriVariables = journalpostIdUriVariables(journalpostId))
+
+    private val journalpostUri =
+        UriComponentsBuilder.fromUri(integrasjonerBaseUrl).pathSegment("api/journalpost").build().toUri()
+    private val dokarkivUri =
+        UriComponentsBuilder.fromUri(integrasjonerBaseUrl).pathSegment("api/dokarkiv").build().toUri()
+
+    fun oppdaterJournalpost(
+        oppdaterJournalpostRequest: OppdaterJournalpostRequest,
+        journalpostId: String,
+        saksbehandler: String?,
+    ): OppdaterJournalpostResponse {
+        return putForEntity<OppdaterJournalpostResponse>(
+            URI.create("$dokarkivUri/$journalpostId").toString(),
```

Comment:
Med "riktigt oppsatt restTemplate" så får vi metrics fra url'er, og for å ikke generere metric per journalpostId så burde vi bruke templates her, 
Ta gjerne en titt på hvordan dette er løst i andre clienter, se eks i oppgaveClient
```kotlin
val uri = UriComponentsBuilder.fromUri(oppgaveUri).pathSegment("{id}").encode().toUriString()

        return getForEntity<Oppgave>(uri, uriVariables = oppgaveIdUriVariables(oppgaveId))
```